include/glow/Runtime/RuntimeTypes.h

```diff
@@ -171,6 +171,30 @@ struct HostConfig {
   size_t executorThreads{3};
 };
 
+/// This is struct for user defined partition.
+struct PartitionConfig {
+  /// The name of the function to be partitioned.
+  std::string funcName;
+  /// The number of user defined partitions.
+  /// The partition ids are between 0 and numOfPartitions - 1, inclusive.
+  size_t numOfPartitions;
+  /// The backend for each partition. backendNames.size() == numOfPartitions.
+  std::vector<std::string> backendNames;
+  /// The name for each partition. partitionNames.size() == numOfPartitions.
+  std::vector<std::string> partitionNames;
+  /// The mapping between nodes' name to Partition ids. Assume there are n nodes
+  /// and m partitions. We have 2 types of valid mapping: 1. all nodes are
+  /// mapped to a partition. 2. For i-th (0 <= i < m) partition, the nodes
+  /// mapped to this partition id are not in this map, and the nodes mapped to
+  /// other partitions ids must be in this map. The node's name should be the
+  /// name in Glow function and may be different from the original name from
+  /// models. Since Glow will mangle names to make them unique.
```

Comment:
It's not only to make them unique. We also e.g. replace / with _. Maybe something else.
You may write: Since Glow will mangle names according to its own conventions.